app/rest/rest/src/main/java/io/syndesis/rest/v1/handler/connection/CustomConnectorHandler.java

```diff
@@ -35,19 +35,22 @@
 import io.swagger.annotations.ApiResponses;
 import io.syndesis.connector.generator.ConnectorSummary;
 import io.syndesis.dao.manager.DataManager;
+import io.syndesis.model.Kind;
 import io.syndesis.model.ListResult;
 import io.syndesis.model.connection.Connector;
 import io.syndesis.model.connection.ConnectorSettings;
 import io.syndesis.rest.util.PaginationFilter;
 import io.syndesis.rest.util.ReflectiveSorter;
+import io.syndesis.rest.v1.operations.Deleter;
+import io.syndesis.rest.v1.operations.Getter;
 import io.syndesis.rest.v1.operations.PaginationOptionsFromQueryParams;
 import io.syndesis.rest.v1.operations.SortOptionsFromQueryParams;
 import io.syndesis.rest.v1.util.PredicateFilter;
 
 import org.springframework.context.ApplicationContext;
 
 @Api(tags = {"custom-connector", "connector-template"})
-public final class CustomConnectorHandler extends BaseConnectorGeneratorHandler {
+public final class CustomConnectorHandler extends BaseConnectorGeneratorHandler implements Getter<Connector>, Deleter<Connector> {
```

Comment:
I'm still not convinced we need this at all. If we do in future, maybe we can add it in, but right now there is no difference between handling get/delete for custom connectors to normal connectors, is there? I don't like having two different endpoints for effectively same functionality.